test/algo/compression/pruning/test_concrete_trace_mmdetection.py

```diff
@@ -25,7 +25,7 @@
     # 'ddod/ddod_r50_fpn_1x_coco',
     # 'deepfashion/mask-rcnn_r50_fpn_15e_deepfashion',
     # 'deformable_detr/deformable-detr_r50_16xb2-50e_coco',
-    'detr/detr_r18_8xb2-500e_coco',
+    # 'detr/detr_r18_8xb2-500e_coco',
```

Comment:
these two nets would give weird output if I applied to `graph.eliminate_dead_code()`.